### DISCUSSION TITLE
Webpack build fails in 4.25.0

### DISCUSSION THREAD

**Discussion by rdk8** [digitalfortress-tech/typeahead-standalone#76] [General]
Hello, thank you for maintaining this project. I am using it in a laravel project that uses mix (webpack) for bundling. The 4.25.0 release breaks the bundling presumably because of the migration from webpack to vite. What are your migration recommendations?

**Comment by niketpathak** [comment]
Hello, thank you for your comment. Normally, there shouldn't be any problem in the bundling (even with webpack). Maybe if you could provide a little more context/details on the error received or a screenshot which would help to diagnose/troubleshoot exactly. 

There are no special steps to follow if you are using ESM to import the library and then build your app (which is your use-case). For using the library in the browser via a CDN, I'd recommend using the UMD version of the library. 

Thanks!

**Reply by rdk8** [reply]
Thanks, I will send details later this week.

**Reply by rdk8** [reply]
I have posted the bug in issue #77
